src/components/Admin/index.jsx

```diff
@@ -45,12 +45,12 @@ class Admin extends React.Component {
     this.props.clearDashboardAnalytics();
   }
 
-  getMetadataForAction(actionSlug) {
+  getMetadataForAction(enterpriseId, actionSlug) {
```

Comment:
`enterpriseId` is already a prop on the `Admin` component. Instead of passing `enterpriseId` in as an function argument, you could retrieve it from props within this function itself:

```
const { enterpriseId } = this.props;
```